src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java

```diff
@@ -70,22 +71,24 @@ public class ModuleIOReal implements ModuleIO {
   // Control modes
   private final VoltageOut driveVoltage = new VoltageOut(0.0).withEnableFOC(true);
   private final VoltageOut turnVoltage = new VoltageOut(0.0).withEnableFOC(true);
-  private final MotionMagicVelocityVoltage drivePIDF =
-      new MotionMagicVelocityVoltage(0.0).withEnableFOC(true);
+  private final VelocityVoltage drivePIDF = new VelocityVoltage(0.0).withEnableFOC(true);
   private final MotionMagicVoltage turnPID = new MotionMagicVoltage(0.0).withEnableFOC(true);
 
   public ModuleIOReal(ModuleConstants constants) {
     name = constants.prefix();
 
-    driveTalon = new TalonFX(constants.driveID());
-    turnTalon = new TalonFX(constants.turnID());
-    cancoder = new CANcoder(constants.cancoderID());
+    driveTalon = new TalonFX(constants.driveID(), "canivore");
+    turnTalon = new TalonFX(constants.turnID(), "canivore");
+    cancoder = new CANcoder(constants.cancoderID(), "canivore");
```

Comment:
public or private?

Reply:
this isnt a declaration its just initialization